File: YourTimesheet/Utils/Messages.cs
using System.Diagnostics.CodeAnalysis;

namespace YourTimesheet.Utils
{
    [ExcludeFromCodeCoverage]
    public class Messages
    {
        public static readonly string UserCantBeAuthorized = "User \"{0}\" can't be authorized";
        public static readonly string EmptyEmailOrPassword = "Email and password can't be empty";
        public static readonly string WrongEmailFormat = "Email has a wrong format";
        public static readonly string ShortPassword = "Password can't be shorter than {0} characters";
        public static readonly string CantCreateUser = "Can't create a user \"{0}\"";
        public static readonly string CantUpdateSettings = "Can't update settings";
        public static readonly string CantUpdatePassword = "Can't update password";
        public static readonly string UserDoesntExist = "User doesn't exist";
        public static readonly string UserDoesntHavePermission = "User doesn't have enough permissions";

        public static readonly string NoItemToRecord = "There is no item to add, delete or update";
        public static readonly string CantUpdateRecord = "Record wasn't updated";

        public static readonly string TimesheetWrongWorkingHoursRange = "Working hours should be in the range {0}..{1}";
        public static readonly string TimesheetWrongDate = "Date can't be empty";
        public static readonly string TimesheetWrongDescription = "Description can't be empty";

        public static readonly string SettingEmptyRole = "User's role can't be empty";
    }
}

File: YourTimesheet/Client/src/app/common/messages.ts
export class Messages {
   public static readonly EmailEmpty: string = 'Email address can\'t be empty';
   public static readonly EmailWrongFormat: string = 'Please provide a correct email address';

   public static readonly PasswordCantBeEmpty = 'Password can\'t be empty';
   public static readonly PasswordWrongLength = 'Password can\'t be shorter than {0} characters';
   public static readonly PasswordDoesntMatch = 'Password does not match the confirmation';

   public static readonly TimesheetItemEmptyDate = 'Date can\'t be empty';
   public static readonly TimesheetItemEmptyDuration = 'Working hours are empty or have a wrong value';
   public static readonly TimesheetItemEmptyDescription = 'Description can\'t be empty';
   public static readonly TimesheetItemAdded = 'Timesheet item successfully added';
   public static readonly TimesheetItemUpdated = 'Timesheet item successfully updated';

   public static readonly TimesheetCantUpdateItem = 'Can\'t update item';
   public static readonly TimesheetCantDeleteItem = 'Can\'t delete item';
   public static readonly TimesheetCantAddItem = 'Can\'t add item';

   public static readonly UserCantBeAuthorized = 'Wrong email or password';
   public static readonly UserIsAuthorized = 'User isn\'t authorized. Please refresh the page.';

   public static readonly ProvideBothFilterDate = 'Please fill both start and end dates or clean them.';

   public static readonly SettingsCantUpdate = 'Can\'t update user settings';
   public static readonly WrongRole = 'Role can\'t be empty';

   public static readonly UserUpdated = 'User successfully updated';
   public static readonly SettingsUpdated = 'Settings successfully updated';
}
